bibliopixel/project/project.py

```diff
@@ -44,10 +45,44 @@ def make_animation(layout, animation, run=None):
     return animation
 
 
-def _make_project(path=None, animation=None, run=None, **kwds):
+def make_remote(layout, remote):
+    OFF_ANIM = {
+        'typename': 'bibliopixel.animation.off.OffAnim'
+    }
+
+    animations = remote.get('animations', None)
+    if not animations or len(animations) == 0:
+        raise ValueError('Must specify `animations` for remote.')
+
+    anim_list = []
+    for anim in animations:
+        run = anim.get('run', {})
+        run['threaded'] = True  # Remote anims must be threaded, maybe a little hacky?
+        anim_obj = dict(anim)
+        anim_obj['animation'] = make_animation(layout, anim['animation'], run)
+        anim_obj.pop('run', None)  # no longer need this
```

Comment:
oh oh... no. That comment is that I no longer need the run entry in that dict, so I'm removing it with pop(). I don't like using the `del` method.